### DISCUSSION TITLE
Cashed files

### DISCUSSION THREAD

**Discussion by Davindex23** [abraunegg/onedrive#2233] [Q&A] [answered]
Hi 
For example my machine is ubuntu and  have only 50 GB storage 
So if i download a file with 70gb sizes, is it possible to write directly on onedrive?(or it should be cashed in hard drive and then uploaded to onedrive)
Srry for newbie question
Thanks

**Comment by abraunegg** [comment, marked as answer]
@Davindex23 
> For example my machine is ubuntu and have only 50 GB storage
> So if i download a file with 70gb sizes, is it possible to write directly on onedrive?

It does not matter what solution you use - a 70GB file needs to be written to disk to modify .. and your disk is too small.

Your only option is to edit the file online, so no download or 'on-demand' local file is used.
